From: Romain Perier <mrpouet@gentoo.org>
Date: Sun, 4 Apr 2010 11:24:07 +0200
Subject: Do not collide with GNOME_FastUserSwicthApplet.server

This file is already installed by gdm, however it's also required by indicator-applet-session
(gdm-user-switch-applet-like but a little different) that's why I renamed it properly.

---
 data/GNOME_FastUserSwitchApplet.server.in.in   |   28 ------------------------
 data/GNOME_IndicatorAppletSession.server.in.in |   28 ++++++++++++++++++++++++
 data/Makefile.am                               |    4 +-
 src/applet-main.c                              |    2 +-
 4 files changed, 31 insertions(+), 31 deletions(-)
 delete mode 100644 data/GNOME_FastUserSwitchApplet.server.in.in
 create mode 100644 data/GNOME_IndicatorAppletSession.server.in.in

--- a/data/GNOME_FastUserSwitchApplet.server.in.in
+++ /dev/null
@@ -1,28 +0,0 @@
-<oaf_info>
-  <oaf_server iid="OAFIID:GNOME_FastUserSwitchApplet_Factory" type="exe"
-              location="@LIBEXECDIR@/indicator-applet-session">
-
-    <oaf_attribute name="repo_ids" type="stringv">
-      <item value="IDL:Bonobo/GenericFactory:1.0"/>
-      <item value="IDL:Bonobo/Unknown:1.0"/>
-    </oaf_attribute>
-    <oaf_attribute name="name" type="string" value="Indicator Applet Factory"/>
-    <oaf_attribute name="description" type="string" value="Indicator Applet Factory"/>
-    <oaf_attribute name="bonobo:environment" type="stringv">
-       <item value="DBUS_SESSION_BUS_ADDRESS"/>
-    </oaf_attribute>
-  </oaf_server>
-
-  <oaf_server iid="OAFIID:GNOME_FastUserSwitchApplet" type="factory"
-              location="OAFIID:GNOME_FastUserSwitchApplet_Factory">
-
-    <oaf_attribute name="repo_ids" type="stringv">
-      <item value="IDL:GNOME/Vertigo/PanelAppletShell:1.0"/>
-      <item value="IDL:Bonobo/Control:1.0"/>
-      <item value="IDL:Bonobo/Unknown:1.0"/>
-    </oaf_attribute>
-    <oaf_attribute name="name" type="string" _value="Indicator Applet Session"/>
-    <oaf_attribute name="description" type="string" _value="A place to adjust your status, change users or exit your session."/>
-    <oaf_attribute name="panel:icon" type="string" value="indicator-applet"/>
-  </oaf_server>
-</oaf_info>
--- /dev/null
+++ b/data/GNOME_IndicatorAppletSession.server.in.in
@@ -0,0 +1,28 @@
+<oaf_info>
+  <oaf_server iid="OAFIID:GNOME_IndicatorAppletSession_Factory" type="exe"
+              location="@LIBEXECDIR@/indicator-applet-session">
+
+    <oaf_attribute name="repo_ids" type="stringv">
+      <item value="IDL:Bonobo/GenericFactory:1.0"/>
+      <item value="IDL:Bonobo/Unknown:1.0"/>
+    </oaf_attribute>
+    <oaf_attribute name="name" type="string" value="Indicator Applet Factory"/>
+    <oaf_attribute name="description" type="string" value="Indicator Applet Factory"/>
+    <oaf_attribute name="bonobo:environment" type="stringv">
+       <item value="DBUS_SESSION_BUS_ADDRESS"/>
+    </oaf_attribute>
+  </oaf_server>
+
+  <oaf_server iid="OAFIID:GNOME_IndicatorAppletSession" type="factory"
+              location="OAFIID:GNOME_IndicatorAppletSession_Factory">
+
+    <oaf_attribute name="repo_ids" type="stringv">
+      <item value="IDL:GNOME/Vertigo/PanelAppletShell:1.0"/>
+      <item value="IDL:Bonobo/Control:1.0"/>
+      <item value="IDL:Bonobo/Unknown:1.0"/>
+    </oaf_attribute>
+    <oaf_attribute name="name" type="string" _value="Indicator Applet Session"/>
+    <oaf_attribute name="description" type="string" _value="A place to adjust your status, change users or exit your session."/>
+    <oaf_attribute name="panel:icon" type="string" value="indicator-applet"/>
+  </oaf_server>
+</oaf_info>
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -9,7 +9,7 @@ serverdir =					\
 server_in_files =				\
 	GNOME_IndicatorApplet.server.in \
 	GNOME_IndicatorAppletComplete.server.in \
-	GNOME_FastUserSwitchApplet.server.in
+	GNOME_IndicatorAppletSession.server.in
 server_DATA =					\
 	$(server_in_files:.server.in=.server)
 
@@ -70,7 +70,7 @@ EXTRA_DIST =					\
 	$(icons_DATA) \
 	GNOME_IndicatorApplet.server.in.in	\
 	GNOME_IndicatorAppletComplete.server.in.in	\
-	GNOME_FastUserSwitchApplet.server.in.in	\
+	GNOME_IndicatorAppletSession.server.in.in	\
 	$(schema_in_files)
 
 MAINTAINERCLEANFILES =				\
--- a/src/applet-main.c
+++ b/src/applet-main.c
@@ -63,7 +63,7 @@ PANEL_APPLET_BONOBO_FACTORY ("OAFIID:GNOME_IndicatorApplet_Factory",
                applet_fill_cb, NULL);
 #endif
 #ifdef INDICATOR_APPLET_SESSION
-PANEL_APPLET_BONOBO_FACTORY ("OAFIID:GNOME_FastUserSwitchApplet_Factory",
+PANEL_APPLET_BONOBO_FACTORY ("OAFIID:GNOME_IndicatorAppletSession_Factory",
                PANEL_TYPE_APPLET,
                "indicator-applet-session", "0",
                applet_fill_cb, NULL);
